Header header
string node_id
int8 priority
string command
